api/v1alpha6/openstackcluster_types.go

```diff
@@ -210,6 +210,8 @@ type OpenStackClusterStatus struct {
 	// and/or logged in the controller's output.
 	// +optional
 	FailureMessage *string `json:"failureMessage,omitempty"`
+
+	Conditions clusterv1.Conditions `json:"conditions,omitempty"`
```

Comment:
You shouldn't modify v1alpha6, only v1beta1 now.

Reply:
get it.thanks